.gitignore

```diff
@@ -6,4 +6,12 @@ coverage/
 build/
 husky.sh
 yarn-debug.log*
-yarn-error.log*
\ No newline at end of file
+yarn-error.log*
+.idea/tasks.xml
+.idea/dictionaries
+.idea/vcs.xml
+.idea/jsLibraryMappings.xml
+.idea/jsLinters
+.idea/prettier.xml
+.idea/workspace.xml
+*.iws
```

Comment:
1. just idea folder is enough
2. remove .idea folder

Reply:
done